### ISSUE TITLE
<STOP> is not a number.

### ISSUE THREAD

**Issue by kwalcock** [clulab/processors#430]
I need to narrow this down quite a bit, but this exception is thrown while the CluProcessor tries to annotate the text of this attached document.  Maybe it looks familiar.  This is processors 8.2.2.  It does have a custom tokenizer added to it.

```
15:57:42.652 [ForkJoinPool-2-worker-1] ERROR o.c.w.e.a.b.ExtractCdrMetaFromDirectory$ - Exception for file ..\corpora\dec2020\input\bad\1101b0d44a26c265a2e7b3d938ea1252.json
java.lang.NumberFormatException: For input string: "<STOP>"
	at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65) ~[na:1.8.0_181]
	at java.lang.Integer.parseInt(Integer.java:580) ~[na:1.8.0_181]
	at java.lang.Integer.parseInt(Integer.java:615) ~[na:1.8.0_181]
	at scala.collection.immutable.StringLike.toInt(StringLike.scala:301) ~[scala-library-2.12.4.jar:na]
	at scala.collection.immutable.StringLike.toInt$(StringLike.scala:301) ~[scala-library-2.12.4.jar:na]
	at scala.collection.immutable.StringOps.toInt(StringOps.scala:29) ~[scala-library-2.12.4.jar:na]
	at org.clulab.processors.clu.CluProcessor.$anonfun$parseSentence$3(CluProcessor.scala:199) ~[processors-main_2.12-8.2.2.jar:8.2.2]
	at scala.runtime.java8.JFunction1$mcVI$sp.apply(JFunction1$mcVI$sp.java:12) ~[scala-library-2.12.4.jar:na]
	at scala.collection.TraversableLike$WithFilter.$anonfun$foreach$1(TraversableLike.scala:789) ~[scala-library-2.12.4.jar:na]
	at scala.collection.immutable.Range.foreach(Range.scala:156) ~[scala-library-2.12.4.jar:na]
	at scala.collection.TraversableLike$WithFilter.foreach(TraversableLike.scala:788) ~[scala-library-2.12.4.jar:na]
	at org.clulab.processors.clu.CluProcessor.$anonfun$parseSentence$1(CluProcessor.scala:198) ~[processors-main_2.12-8.2.2.jar:8.2.2]
	at org.clulab.processors.clu.CluProcessor.$anonfun$parseSentence$1$adapted(CluProcessor.scala:193) ~[processors-main_2.12-8.2.2.jar:8.2.2]
	at scala.collection.immutable.Range.foreach(Range.scala:156) ~[scala-library-2.12.4.jar:na]
	at org.clulab.processors.clu.CluProcessor.parseSentence(CluProcessor.scala:193) ~[processors-main_2.12-8.2.2.jar:8.2.2]
	at org.clulab.processors.clu.CluProcessor.$anonfun$parse$1(CluProcessor.scala:419) ~[processors-main_2.12-8.2.2.jar:8.2.2]
	at scala.collection.IndexedSeqOptimized.foreach(IndexedSeqOptimized.scala:32) ~[scala-library-2.12.4.jar:na]
	at scala.collection.IndexedSeqOptimized.foreach$(IndexedSeqOptimized.scala:29) ~[scala-library-2.12.4.jar:na]
	at scala.collection.mutable.ArrayOps$ofRef.foreach(ArrayOps.scala:191) ~[scala-library-2.12.4.jar:na]
	at org.clulab.processors.clu.CluProcessor.parse(CluProcessor.scala:418) ~[processors-main_2.12-8.2.2.jar:8.2.2]
	at org.clulab.processors.clu.CluProcessor.annotate(CluProcessor.scala:91) ~[processors-main_2.12-8.2.2.jar:8.2.2]
	at org.clulab.processors.Processor.annotate(Processor.scala:65) ~[processors-main_2.12-8.2.2.jar:8.2.2]
	at org.clulab.processors.Processor.annotate$(Processor.scala:62) ~[processors-main_2.12-8.2.2.jar:8.2.2]
	at org.clulab.processors.clu.CluProcessor.annotate(CluProcessor.scala:22) ~[processors-main_2.12-8.2.2.jar:8.2.2]
	at org.clulab.wm.eidos.groundings.grounders.SRLCompositionalGrounder.groundEidosMention(SRLCompositionalGrounder.scala:106) ~[classes/:na]
	at org.clulab.wm.eidos.groundings.OntologyHandler.$anonfun$ground$1(OntologyHandler.scala:38) ~[classes/:na]
	at scala.collection.TraversableLike.$anonfun$flatMap$1(TraversableLike.scala:241) ~[scala-library-2.12.4.jar:na]
	at scala.collection.immutable.List.foreach(List.scala:389) ~[scala-library-2.12.4.jar:na]
	at scala.collection.TraversableLike.flatMap(TraversableLike.scala:241) ~[scala-library-2.12.4.jar:na]
	at scala.collection.TraversableLike.flatMap$(TraversableLike.scala:238) ~[scala-library-2.12.4.jar:na]
	at scala.collection.immutable.List.flatMap(List.scala:352) ~[scala-library-2.12.4.jar:na]
	at org.clulab.wm.eidos.groundings.OntologyHandler.ground(OntologyHandler.scala:36) ~[classes/:na]
	at org.clulab.wm.eidos.EidosSystem.$anonfun$mkEidosRefiners$2(EidosSystem.scala:85) ~[classes/:na]
	at org.clulab.wm.eidos.EidosSystem.$anonfun$mkEidosRefiners$2$adapted(EidosSystem.scala:85) ~[classes/:na]
	at scala.collection.immutable.Stream.foreach(Stream.scala:530) ~[scala-library-2.12.4.jar:na]
	at org.clulab.wm.eidos.EidosSystem.$anonfun$mkEidosRefiners$1(EidosSystem.scala:85) ~[classes/:na]
	at org.clulab.wm.eidos.EidosSystem.$anonfun$refineEidosMentions$2(EidosSystem.scala:167) ~[classes/:na]
	at org.clulab.wm.eidos.utils.Timer$.time(Timer.scala:77) ~[classes/:na]
	at org.clulab.wm.eidos.EidosSystem.$anonfun$refineEidosMentions$1(EidosSystem.scala:166) ~[classes/:na]
	at scala.collection.LinearSeqOptimized.foldLeft(LinearSeqOptimized.scala:122) ~[scala-library-2.12.4.jar:na]
	at scala.collection.LinearSeqOptimized.foldLeft$(LinearSeqOptimized.scala:118) ~[scala-library-2.12.4.jar:na]
	at scala.collection.immutable.List.foldLeft(List.scala:86) ~[scala-library-2.12.4.jar:na]
	at org.clulab.wm.eidos.EidosSystem.refineEidosMentions(EidosSystem.scala:165) ~[classes/:na]
	at org.clulab.wm.eidos.EidosSystem.extractFromDoc(EidosSystem.scala:182) ~[classes/:na]
	at org.clulab.wm.eidos.EidosSystem.extractFromDoc(EidosSystem.scala:195) ~[classes/:na]
	at org.clulab.wm.eidos.EidosSystem.extractFromText(EidosSystem.scala:216) ~[classes/:na]
	at org.clulab.wm.eidos.apps.batch.ExtractCdrMetaFromDirectory$.$anonfun$new$4(ExtractCdrMetaFromDirectory.scala:55) [classes/:na]
	at scala.runtime.java8.JFunction0$mcI$sp.apply(JFunction0$mcI$sp.java:12) ~[scala-library-2.12.4.jar:na]
	at org.clulab.wm.eidos.utils.Timer.time(Timer.scala:13) ~[classes/:na]
	at org.clulab.wm.eidos.apps.batch.ExtractCdrMetaFromDirectory$.$anonfun$new$3(ExtractCdrMetaFromDirectory.scala:49) [classes/:na]
	at org.clulab.wm.eidos.apps.batch.ExtractCdrMetaFromDirectory$.$anonfun$new$3$adapted(ExtractCdrMetaFromDirectory.scala:44) [classes/:na]
	at scala.collection.parallel.mutable.ParArray$ParArrayIterator.foreach(ParArray.scala:142) ~[scala-library-2.12.4.jar:na]
	at scala.collection.parallel.ParIterableLike$Foreach.leaf(ParIterableLike.scala:970) ~[scala-library-2.12.4.jar:na]
	at scala.collection.parallel.Task.$anonfun$tryLeaf$1(Tasks.scala:49) ~[scala-library-2.12.4.jar:na]
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12) ~[scala-library-2.12.4.jar:na]
	at scala.util.control.Breaks$$anon$1.catchBreak(Breaks.scala:63) ~[scala-library-2.12.4.jar:na]
	at scala.collection.parallel.Task.tryLeaf(Tasks.scala:52) ~[scala-library-2.12.4.jar:na]
	at scala.collection.parallel.Task.tryLeaf$(Tasks.scala:46) ~[scala-library-2.12.4.jar:na]
	at scala.collection.parallel.ParIterableLike$Foreach.tryLeaf(ParIterableLike.scala:967) ~[scala-library-2.12.4.jar:na]
	at scala.collection.parallel.AdaptiveWorkStealingTasks$WrappedTask.compute(Tasks.scala:149) ~[scala-library-2.12.4.jar:na]
	at scala.collection.parallel.AdaptiveWorkStealingTasks$WrappedTask.compute$(Tasks.scala:145) ~[scala-library-2.12.4.jar:na]
	at scala.collection.parallel.AdaptiveWorkStealingForkJoinTasks$WrappedTask.compute(Tasks.scala:440) ~[scala-library-2.12.4.jar:na]
	at java.util.concurrent.RecursiveAction.exec(RecursiveAction.java:189) ~[na:1.8.0_181]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289) ~[na:1.8.0_181]
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056) ~[na:1.8.0_181]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692) ~[na:1.8.0_181]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157) ~[na:1.8.0_181]
```
[1101b0d44a26c265a2e7b3d938ea1252.json.txt](https://github.com/clulab/processors/files/5432081/1101b0d44a26c265a2e7b3d938ea1252.json.txt)

**Comment by MihaiSurdeanu** [comment]
Hi @kwalcock : this has been fixed in issue #424. Has the parallel branch not been merged yet?

**Comment by kwalcock** [comment]
It did look like a familiar message.  I forgot that the fix was limited to the parallel branch.  I think that one still needs a bit of work around the initialization routines before everything is able to use it.  I am going to copy just the bug fix into a local branch and work on the other later.
